packages/wrangler/src/__tests__/api/startDevWorker/LocalRuntimeController.test.ts

```diff
@@ -138,9 +138,10 @@ function configDefaults(
 }
 
 describe("LocalRuntimeController", () => {
-	const teardown = useTeardown();
 	mockConsoleMethods();
 	runInTempDir();
+	// Make sure teardown is declared after runInTempDir so it runs before we delete the temp directory
+	const teardown = useTeardown();
```

Comment:
Also, this was causing teardown logging to be displayed in the tests, since the console mocking was being undone before the controller was being torn down